src/System.Threading.Overlapped/src/System.Threading.Overlapped.builds

```diff
@@ -10,11 +10,7 @@
     </Project>
     <Project Include="System.Threading.Overlapped.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>net46</TargetGroup>
-    </Project>
-    <Project Include="System.Threading.Overlapped.csproj">
```

Comment:
We've had different opinions in the past about what direction to take in here, but in general we have been removing them and just keeping the latest for now, so this looks fine to me.